### ISSUE TITLE
How to start Learning Data Science as a beginner?

### ISSUE THREAD

**Issue by mjbahmani** [mjbahmani/10-steps-to-become-a-data-scientist#2]
Hi Every one,
I hope you are all well.
If you have any problem with the repository or have any idea, I am here to hear from you
Thanks for considering this,
Best Regard,
MJ Bahmani

**Comment by Rockyrod** [comment]
Thanks MJ for the great work.
This is a platform i found where i can brush-up my skills.
Thanks again...

Vikash Prasad

**Comment by mjbahmani** [comment]
Hi Vikash,
Hi hope it will be useful for you 
😊👍

**Comment by Rockyrod** [comment]
Yes it is going to help me to brush up my data science skills and the way you explained is just awesome 👍

**Comment by thelamehacker** [comment]
Thank you for the efforts. This looks like a great start. I will run through all of your tutorials. :)

**Comment by CarlosCross** [comment]
Hello Bahmani, never is late to start learning... I'm going to start with this material...

**Comment by mjbahmani** [comment]
Hey there,
Great to hear that!
I am ready to answer your Questions about the course!

**Comment by CarlosCross** [comment]
Thank you... I don't know why I can't open these files from Jupyter :( Do you have some idea? because I want to take advantage of studying offline.... thanks

![image](https://user-images.githubusercontent.com/22893785/59951694-ee612700-9436-11e9-94a5-db8603d31e85.png)

**Comment by CarlosCross** [comment]
This is the error I have received:

![image](https://user-images.githubusercontent.com/22893785/59951809-3da75780-9437-11e9-972c-8dc7c22e7eba.png)

**Comment by CarlosCross** [comment]
I'm sorry, was my bad using GitHub... Just RAW option...

**Comment by mjbahmani** [comment]
Did you test this file in online jupyter notebook like Colab?
